dotCMS/src/main/java/com/dotcms/util/pagination/TemplateView.java

```diff
@@ -0,0 +1,410 @@
+        return selectedimage;
+    }
+
+    public String getImage() {
+        return image;
+    }
+
+    public boolean isDrawed() {
+        return drawed;
+    }
+
+    public boolean isDeleted() {
+        return deleted;
+    }
+
+    public boolean isLive() {
+        return live;
+    }
+
+    public boolean isLocked() {
+        return locked;
+    }
+
+    public boolean isWorking() {
+        return working;
+    }
+
+    public boolean isNew() {
+        return isNew;
+    }
+
+    public boolean isHasLiveVersion() {
+        return hasLiveVersion;
+    }
+
+    public boolean isShowOnMenu() {
+        return showOnMenu;
+    }
+
+    public String getDrawedBody() {
+        return drawedBody;
+    }
+
+    public int getCountAddContainer() {
+        return countAddContainer;
+    }
+
+    public int getCountContainers() {
+        return countContainers;
+    }
+
+    public String getHeadCode() {
+        return headCode;
+    }
+
+    public String getTheme() {
+        return theme;
+    }
+
+    public String getThemeName() {
+        return themeName;
+    }
+
+    public String getFooter() {
+        return footer;
+    }
+
+    public String getFriendlyName() {
+        return friendlyName;
+    }
+
+    public String getHeader() {
+        return header;
+    }
+
+    public Date getModDate() {
+        return modDate;
+    }
+
+    public String getModUser() {
+        return modUser;
+    }
+
+    public String getOwner() {
+        return owner;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public int getSortOrder() {
+        return sortOrder;
+    }
+
+    public boolean isCanRead() {
+        return canRead;
+    }
+
+    public boolean isCanWrite() {
+        return canWrite;
+    }
+
+    public boolean isCanPublish() {
+        return canPublish;
+    }
+
+    public static final class Builder {
+
+        private  String identifier;
+        private  String inode;
+        private  String categoryId;
+        private  String body;
+        private  String selectedimage;
+        private  String image;
+        private  boolean drawed;
+        private  boolean deleted;
+        private  boolean live;
+        private  boolean locked;
+        private  boolean working;
+        private  boolean isNew;
+        private  boolean hasLiveVersion;
+        private  boolean showOnMenu;
+        private  String drawedBody;
+        private  int countAddContainer;
+        private  int countContainers;
+        private  String headCode;
+        private  String theme;
+        private  String themeName;
+        private  String footer;
+        private  String friendlyName;
+        private  String header;
+        private  Date modDate;
+        private  String modUser;
+        private  String owner;
+        private  String name;
+        private  String title;
+        private  int sortOrder;
```

Comment:
Codacy found an issue: [Field sortOrder has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6042990)